requirements/constraints.txt

```diff
@@ -1,5 +1,7 @@
 awscli==1.29.45
 black==22.10.0
+# urllib3 major version 2, released on May 4th 2023, breaks botocore used
+# by awscli (removed DEFAULT_CIPHERS list from urllib3.util.ssl_)
```

Comment:
should the comment be above urllib3 instead?

Reply:
This comment justifies the version pinning, so it should be above the version pinning hence in `constraints.txt` IMO. WDYT?

Reply:
Oops mb got you ! Fixed in 9fa38ba